code_snippets/api-events-stream.rb

```diff
@@ -6,7 +6,7 @@
 
 dog = Dogapi::Client.new(api_key, app_key)
 
-start_time = Time.now.to_i
-end_time = Time.now.to_i + 100
+end_time = Time.now.to_i 
```

Comment:
Looks like an extra space at the end here, but once you delete that, it's good to merge.